### PR TITLE
Feature connect sign in form

### PR DESCRIPTION
**Pull request by IgorPopach**

### CODE REVIEW

File: client/src/components/SignIn/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    <div className="form-group">
+                        <input type="password" name="password" 
+                        placeholder="Password"
+                        value={ this.state.password }
+                        onChange={this.handleInputChange}
+                        />
+                    </div>
+                    <div className="form-group">
+                        <button className="btn btn-lg btn-secondary btn-block" 
+                        type="submit">LOGIN</button>
```
Comment (YuliaFursovych) [inline client/src/components/SignIn/SignIn.js]: Don't forget to rename LOGIN to Sign In.

File: client/src/components/SignIn/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            email,
+            password,
+        }
+        loginUser(user);
+    }
+    render() {
+        console.log('this.props', this.props)
+        return (
+            <div className="container">
+                <h2 style={{marginBottom: '40px', color: 'white'}}>Please sign in</h2>
```
Comment (YuliaFursovych) [inline client/src/components/SignIn/SignIn.js]: It should be better to write styles in the style file, even if they are there temporarily. Especially if this file has already been created.

File: client/src/components/SignIn/SignIn.js (hunk 1 above)
Comment (IgorPopach) [reply client/src/components/SignIn/SignIn.js]: Ok! I'll fix it...

File: client/package.json [hunk 3]
```diff
@@ -31,5 +31,6 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "proxy": "http://localhost:3001"
```
Comment (YuliaFursovych) [inline client/package.json:39]: I've got error: "Can't resolve axios and jwt-decode". Do we need to add them to project dependencies?

Comment (IgorPopach) [reply client/package.json:39]: Yep! Please run npm I axios and jwt-decode! I lost them after merging from dev(((
I'll added them before merging...

File: client/src/components/SignIn/SignIn.js (hunk 2 above)
Comment (IgorPopach) [reply client/src/components/SignIn/SignIn.js]: Ok, thanks

File: client/src/components/SignIn/SignIn.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+
+import { loginUser } from './../../authentication';
+
+class Login extends Component {
```
Comment (YuliaFursovych) [inline client/src/components/SignIn/SignIn.js:9]: When you input something in the email field, the password field is empty and you try to sign in, it does not indicate an error that the password field is empty. If on the contrary, then there is a error. Should we prevent that?
Also, there are no messages from the server that let you know whether are sign in successfully or not.

**Pull request closed without merging**